src/utils.ts

```diff
@@ -18,6 +18,14 @@ export function toWikiLink(input: string): string {
     .replace(/\)/g, "%29")}`;
 }
 
+export function toKoldbLink(input: string): string {
+  return `koldb.com/player.php?name=${encodeURI(input.replace(/\s/g, "%20"))}`;
```

Comment:
using a regexp in string.replace means it automatically does a replaceAll, right?

Reply:
With g flag yeah